docs/UserGuide.md

```diff
@@ -79,12 +79,20 @@ Tag food item - tag <Index> /with <Tag Name>
 
 ### Adding a food item: `add`
 
-*placeholder*
+Add food items into your fridge.
 
-Format: `placeholder`
+Format: `add`
 
 Examples:
-* `placeholder`
+* `add Spinach`
+* `add Meiji Milk`
+* `add Awfully Chocolate`
```

Comment:
I think can remove this since there is already an example section below, other than that should it looks good

Reply:
removed items